src/matching/matcher.ts

```diff
@@ -17,11 +17,6 @@ export class PairMatcher {
     // useful for text objects.
     "<" : { match: ">",  nextMatchIsForward: true  },
     ">" : { match: "<",  nextMatchIsForward: false },
-    // These are useful for deleting closing and opening quotes, but don't seem to negatively
-    // affect how text objects such as `ci"` work, which was my worry.
-    '"' : { match: '"',  nextMatchIsForward: false  },
```

Comment:
sure, but you shouldn't be using quotes in the pairmatcher for surround...there is no open or close, so how is doing a surround with " different than a surround with any other normal character?